### PR TITLE
Modify `LIST_VALUE_CLAUSE` and ` PHYSICAL_ATTRIBUTE_CLAUSE`

### PR DESCRIPTION
**Pull request by ani-sha**

### CODE REVIEW

File: sql/plsql/PlSqlParser.g4 [hunk 1]
```diff
@@ -992,6 +992,21 @@ validation_clauses
...
+compute_clauses
+    : COMPUTE SYSTEM? STATISTICS for_clause?
+    ;
+
+for_clause
+    : FOR TABLE (for_clause)*
+    | FOR ALL INDEXED? COLUMNS (SIZE UNSIGNED_INTEGER)? (for_clause)*
+    | FOR COLUMNS (SIZE UNSIGNED_INTEGER)? for_column_clauses (for_clause)*
+    | FOR ALL LOCAL? INDEXES
+    ;
```
Comment (KvanTTT) [inline sql/plsql/PlSqlParser.g4:1004]: I suggest factoring and removing redundant parentheses:

```suggestion
for_clause
    : FOR ( TABLE for_clause*
          | ALL (INDEXED? COLUMNS (SIZE UNSIGNED_INTEGER)? for_clause* | LOCAL? INDEXES)
          | COLUMNS (SIZE UNSIGNED_INTEGER)? for_column_clauses for_clause*
          )
    ;
```

File: sql/plsql/PlSqlParser.g4 [hunk 2]
```diff
@@ -992,6 +992,21 @@ validation_clauses
...
+
+for_column_clauses
+    : column_name (SIZE UNSIGNED_INTEGER) (for_column_clauses)*
```
Comment (KvanTTT) [inline sql/plsql/PlSqlParser.g4:1007]: No need to use parentheses here:

```suggestion
    : column_name SIZE UNSIGNED_INTEGER for_column_clauses*
```

File: sql/plsql/PlSqlParser.g4 [hunk 3]
```diff
@@ -992,6 +992,21 @@ validation_clauses
...
+for_clause
+     : FOR ( TABLE for_clause*
+           | ALL (INDEXED? COLUMNS (SIZE UNSIGNED_INTEGER)? for_clause* | LOCAL? INDEXES)
+           | COLUMNS (SIZE UNSIGNED_INTEGER)? for_column_clauses for_clause*
+           )
+    ;
+
+for_column_clauses
+    : column_name SIZE UNSIGNED_INTEGER for_column_clauses*
+    ;
```
Comment (KvanTTT) [inline sql/plsql/PlSqlParser.g4:1008]: What do you think about simplification by the following way?

```suggestion
for_clause
     : FOR ( TABLE for_clause*
           | ALL (INDEXED? COLUMNS (SIZE UNSIGNED_INTEGER)? for_clause* | LOCAL? INDEXES)
           | COLUMNS (SIZE UNSIGNED_INTEGER)? (column_name SIZE UNSIGNED_INTEGER)+ for_clause*
           )
    ;
```

It looks like there is no need for separated `for_column_clauses` rule.

Comment (ani-sha) [reply sql/plsql/PlSqlParser.g4]: That makes sense @KvanTTT. Simplified the changes.

**Pull request merged**